configure.ac

```diff
@@ -91,9 +94,13 @@ AC_SEARCH_LIBS([event_base_dispatch], [event],, [missing_libevent=1],
                [$save_LIBS])
 AC_SEARCH_LIBS([evthread_use_pthreads], [event_pthreads],, [missing_libevent=1],
                [$save_LIBS])
+AC_SEARCH_LIBS([bufferevent_openssl_socket_new], [event_openssl],, [missing_libevent_openssl=1],
+               [$save_LIBS])
```

Comment:
I'll let it slide because I know that's coming right next, but ideally, you wouldn't have added this SSL stuff that goes completely unused in this pull request (same story in `Makefile.am`). :wink: